src/main/java/mate/academy/springboot/swagger/service/mapper/ProductRequestDtoMapper.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.springboot.swagger.service.mapper;
+
+import mate.academy.springboot.swagger.dto.ProductRequestDto;
+import mate.academy.springboot.swagger.model.Product;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductRequestDtoMapper implements RequestDtoMapper<ProductRequestDto, Product> {
```

Comment:
can we combine this class with `ProductResponseDtoMapper` and name it `ProductDtoMapper`?